### PR TITLE
LPS-72661 Provide soy capability

### PR DESCRIPTION
**Pull request by jkappler**
cc:// @jbalsas

### CODE REVIEW

**Comment by ealonso** [comment]
Merged!! Thanks!!

File: journal-web/bnd.bnd [hunk 1]
```diff
@@ -12,7 +12,10 @@ Liferay-Releng-Module-Group-Title: Web Content
 Provide-Capability:\
 	liferay.resource.bundle;\
 		resource.bundle.aggregate:String="(bundle.symbolic.name=com.liferay.journal.lang)";\
-		resource.bundle.base.name="content.Language"
+		resource.bundle.base.name="content.Language",\
+	soy;\
+		type="portlet-configuration-css";\
```
Comment (jbalsas) [inline journal-web/bnd.bnd]: Hey @jkappler, @ealonso, this should be `type="journal-web"`, not `configuration-css`...

Comment (ealonso) [inline journal-web/bnd.bnd]: Thansk!!

Comment (ealonso) [inline journal-web/bnd.bnd]: Thanks!!

**Pull request closed without merging**